tests/test_table_transformations.py

```diff
@@ -296,6 +296,96 @@ def test_diagnoseless_biosources(self):
         expected_df.set_index(['Subject Id', 'Diagnosis Id', 'Biosource Id'], inplace=True)
         pdt.assert_frame_equal(df, expected_df)
 
+    def test_result_data_shape_patient_radiology(self):
+        self.test_data = [
```

Comment:
I see the `self.test_data` also being used in other tests, but I don't understand why. Seems to be used as a local variable.